rest_specification/locking.md

```diff
@@ -0,0 +1,7 @@
+# Metadata
+
+* Descriptive name is the Header
+* First paragraph is a short description of the API
+* Request section contains info on what the request looks like
+* Response section contains info on what the response looks like, including errors where applicable
```

Comment:
Followed the same pattern as the other topics. This is a copy of metadata.md as you can see from the title. Fixed that btw.